server/src/api/listings.ts

```diff
@@ -37,4 +37,19 @@ listingRouter.get(
   }
 );
 
+listingRouter.get(
```

Comment:
Could we have a bit of documentation on the API functions? For example, let's say iIf I were a new user of this API, how shall I look up the right listingId in order to correctly call this API?

Reply:
@xlai20 What do you mean by "look up the right listingId"? Also, where would you prefer this documentation to be? I was thinking maybe we can just edit our design doc section on API endpoints so we don't have to change it in many places.

Reply:
What defines as a listing id? Is it just a self-invented primary key?

The documentation should be in this file, or directly on top of the API method.

Reply:
@xlai20 Oh the listing id is the default primary key generated by datastore when we a new entity is created. Ok I understand what you want better now, will add it in!